detox/src/devices/runtime/drivers/ios/IosDriver.js

```diff
@@ -1,41 +0,0 @@
-// @ts-nocheck
```

Comment:
Well I'm not strongly against that but I must in fact vote for restoring this class

Reply:
Resolved in bea03863